lib/logstash/inputs/eventlog.rb

```diff
@@ -21,120 +22,62 @@ class LogStash::Inputs::EventLog < LogStash::Inputs::Base
   default :codec, "plain"
 
   # Event Log Name
-  config :logfile, :validate => :array, :default => [ "Application", "Security", "System" ]
+  config :logfile, :validate => :string, :validate => [ "Application", "Security", "System" ], :default => "Application"
```

Comment:
I think that a combination of catching and propertly reporting this error + that link in a comment will help the users :+1: